frontend/src/apiMocks/mockData.ts

```diff
@@ -0,0 +1,148 @@
+import { CustomAliasData, RandomAliasData } from "../hooks/api/aliases";
+import { ProfileData } from "../hooks/api/profile";
+
+export const mockIds = ["empty", "onboarding", "some", "full"] as const;
```

Comment:
5. The available mockIds are listed here. By enumerating them here, we can ensure that every handler has data to return for every listed ID. (Also, you can append `?mockId=<mockId>` to a request to automatically fake sign in as the given user.)